mlflow/server/js/src/experiment-tracking/components/CompareRunBox.js

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Row, Col, Select } from 'antd';
+import { Typography } from '@databricks/design-system';
+import { RunInfo } from '../sdk/MlflowMessages';
+import { LazyPlot } from './LazyPlot';
+
+const { Option, OptGroup } = Select;
+
+export const CompareRunBox = ({ runUuids, runInfos, metricLists, paramLists }) => {
+  const [xAxis, setXAxis] = useState({ key: undefined, isParam: undefined });
+  const [yAxis, setYAxis] = useState({ key: undefined, isParam: undefined });
```

Comment:
```suggestion
  const [xAxis, setXAxis] = useState();
  const [yAxis, setYAxis] = useState();
```

Reply:
@harupy 
I have tested this locally and found that if the keys are left undefined, there will be key errors.